apps/api/src/app/domain-modules/web-of-knowledge/web-of-knowledge.module.ts

```diff
@@ -21,6 +23,8 @@ import { PersistenceModule } from '../../../persistence/persistence.module';
         GrantResourceReadAccessToUserCommandHandler,
         PublishResource,
         PublishResourceCommandHandler,
+        ProvideAdditionalCreditsForResource,
+        ProvideAdditionalCreditsForResourceCommandHandler,
```

Comment:
Nice, I thought it might come down to adding this to the module, I'm slowly learning!